src/cms/config.js

```diff
@@ -0,0 +1,136 @@
+function generateGuid() {
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
+    var r = (Math.random() * 16) | 0,
+      v = c == 'x' ? r : (r & 0x3) | 0x8;
+    return v.toString(16);
+  });
+}
```

Comment:
Move this to another location.

Lets keep `config.js` to configuration stuff only

Reply:
Done - see latest pull request